<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessesDialog</class>
 <widget class="QWidget" name="ProcessesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ProcessesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
